test/e2e/util/test_kubernetes.go

```diff
@@ -72,15 +72,13 @@ func (k TestKubernetes) NewNamespace(namespace string) {
 		},
 	}
 
-	ns := types.NamespacedName{Name: namespace, Namespace: namespace}
-	err := k.Kubernetes.Client.Get(context.TODO(), ns, obj)
+	err := k.Kubernetes.Client.Get(context.TODO(), types.NamespacedName{Name: namespace}, obj)
 	if err != nil && errors.IsNotFound(err) {
 		err = k.Kubernetes.Client.Create(context.TODO(), obj)
 		ExpectNoError(err)
-		return
+	} else {
+		ExpectNoError(err)
```

Comment:
Why can't `ExpectNoError(err)` call live after `if` block since it's called in both branches? It looks like it could save few lines of code.

Reply:
Yes, sure, will return back. Looks like hit some regression not related to this fix